src/adaptors/apeswap-amm/index.js

```diff
@@ -249,39 +242,145 @@ const apy = async () => {
     )
       .div(1e18)
       .toString();
-
     const lpFees7D =
       (volumesMap[pool.lpToken.toLowerCase()] || []).reduce(
         (acc, { dailyVolumeUSD }) => acc + Number(dailyVolumeUSD),
         0
-      ) * FEE_RATE;
+      ) * CHAINS[chain].feeRate;
     const apyBase = ((lpFees7D * WEEKS_PER_YEAR) / lpReservesUsd) * 100;
-
+    const banana = CHAINS[chain].banana;
     const apyReward = calculateApy(
       poolInfo,
       totalAllocPoint,
       normalizedbananaPerBlock,
-      tokensPrices[BANANA.toLowerCase()],
-      masterChefReservesUsd
+      tokensPrices[banana.toLowerCase()],
+      masterChefReservesUsd,
+      CHAINS[chain].block.year,
+      chain
     );
 
     return {
-      pool: pool.lpToken,
-      chain: utils.formatChain('binance'),
+      pool: `${pool.lpToken}-${chain}`.toLowerCase(),
```

Comment:
hey. i know this is whats in the readme, but its mainly for new adapters. in your case please do not modify the `pool` values. so lets revert this specific change (and all other changes to the pool field if any)